frontend/src/pages/articles/articles-feed.tsx

```diff
@@ -11,13 +14,35 @@ import { MOCKED_REACTIONS, MOCKED_TAGS } from './libs/constants.js';
 const ArticlesFeed: React.FC = () => {
   const dispatch = useAppDispatch();
   const { articles } = useAppSelector(({ articles }) => articles);
+  const { hasMore, load } = usePagination();
+
+  const handleLoadArticles = useCallback(() => {
+    void load(async (skip: number, take: number) => {
+      const data = await dispatch(
+        articlesActions.fetchAll({
+          take,
+          skip,
+        }),
+      ).unwrap();
+
+      return skip + data.items.length < data.total;
```

Comment:
Won't it skip actually existing data to fetch?
Like if first 10 items is already loaded - `skip=10`, but total results count is `18`
Can't we just check if anything were fetched to make decision on `hasMore`?